.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: go
+
+branches:
+  only:
+  - gh-pages
+  - /.*/
+
+service:
+  - docker
+
+script:
+  - docker run --rm -it sqlflow/gohive:dev /build_and_test.bash
```

Comment:
Should we mount the pull request branch into the Docker container?

Reply:
Oh. my god. Yes, I am a fool